march_foot_position_finder:
  ros__parameters:
    foot_gap: 0.25
    step_distance: 0.30

    outlier_distance: 0.05
    sample_size: 3
    height_zero_threshold: 0.02

    foot_width: 0.10
    foot_length: 0.20
    actual_foot_length: 0.33

    derivative_threshold: 0.02
    available_points_ratio: 0.90
    max_z_distance: 0.25
    num_track_points: 30

    displacements_outside: 0.05
    displacements_inside: 0.10
    displacements_near: 0.25
    displacements_far: 0.10
